data/json/items/ranged.json

```diff
@@ -2117,11 +2117,12 @@
     "recoil": 30,
     "durability": 8,
     "burst": 5,
-    "clip_size": 20,
-    "reload": 550,
+    "reload": 100,
     "loudness" : 150,
     "barrel_length" : 4,
-    "valid_mod_locations": [[ "accessories", 4 ],[ "barrel", 1 ],[ "bore", 1 ],[ "conversion", 1 ],[ "grip", 1 ],[ "magazine", 2 ],[ "mechanism", 4 ],[ "muzzle", 1 ],[ "rail", 1 ],[ "sights", 1 ],[ "stock", 1 ],[ "underbarrel", 1 ]]
+    "magazines": [ "m1918mag", "m1918mag_30" ],
+    "valid_mod_locations": [[ "accessories", 4 ],[ "barrel", 1 ],[ "bore", 1 ],[ "conversion", 1 ],[ "grip", 1 ],[ "mechanism", 4 ],[ "muzzle", 1 ],[ "rail", 1 ],[ "sights", 1 ],[ "stock", 1 ],[ "underbarrel", 1 ]],
+    "default_mods": [ "m1918mag" ]
```

Comment:
Don't specify defaults as this support is likely to be deprecated. Please be creative instead with `item_groups` to ensure the magazine is common or rare.

A future PR will likely implement a percentage chance to spawn an a magazine which would make this redundant.